src/components/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import cn from 'classnames';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  filter: Filter;
+  setFilter: React.Dispatch<React.SetStateAction<Filter>>;
+  activeCount: number;
+  completedCount: number;
+  onClearCompleted: () => Promise<void>;
+};
+
+export const Footer: React.FC<Props> = ({
+  setFilter,
+  filter,
+  activeCount,
+  completedCount,
+  onClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeCount} item${activeCount === 1 ? '' : 's'} left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filter === Filter.all,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter(Filter.all)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: filter === Filter.active,
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilter(Filter.active)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={cn('filter__link', {
+            selected: filter === Filter.completed,
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilter(Filter.completed)}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method